.gitignore

```diff
@@ -18,7 +18,9 @@ node_modules/
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-
+client/yarn.lock
+client/package-lock.json
```

Comment:
We don't want to ignore the lock files. https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5?rq=1